src/client/App.tsx

```diff
@@ -49,7 +49,7 @@ const App = () => {
           <MainLayout>
             <Switch>
               <Route exact={true} path="/tutorial" component={Tutorial} />
-              <Route path="/*" component={Home} />
+              <Route path={[`/iotx-${base.chainToken.key}`, `/${base.chainToken.key}-iotx`]} component={Home} />
```

Comment:
Work not well, can not use '/eth-iotx' any more.

Reply:
done